docs/baking.md

```diff
@@ -105,50 +114,27 @@ In order to import the snapshot, run the following command:
 sudo -u tezos tezos-node-<network> snapshot import <path to the snapshot file>
 ```
 
-### Starting the node
+## Setting up baker and endorser key
 
-After the snapshot import, you can finally start the node by running:
+Note that account activation from JSON file and baker registering require
```

Comment:
I would prioritize (that is - mention first) information that is relevant for mainnet: 1) import key from ledger nano s 2) import secret key directly 3) generate new key pair, then fund + delegate 4) use remote signer, with details for 3 and 4 moved to "Advanced" section at the end.